src/test/java/io/confluent/examples/streams/ExampleTestUtils.java

```diff
@@ -0,0 +1,20 @@
+package io.confluent.examples.streams;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.util.Random;
+
+public class ExampleTestUtils {
+
+    public static int randomFreeLocalPort() throws IOException {
+        ServerSocket s = new ServerSocket(0);
+        int port = s.getLocalPort();
+        s.close();
+        return port;
+    }
+
+    public static String randomValidHost() {
+        Random r = new Random();
+        return "127." + r.nextInt(10) + "." + r.nextInt(10) + "." + r.nextInt(256);
```

Comment:
Yay for test fuzzing!

Should we generate `"localhost"` also with some probability?